compiler/codegen/symbol.cpp

```diff
@@ -877,22 +877,42 @@ GenRet ArgSymbol::codegen() {
 
 // If there is a known .pxd file translation for this type, use that.
 // Otherwise, use the normal cname
-static std::string getPXDTypeName(Type* type) {
+static std::string getPythonTypeName(Type* type, bool pxd) {
```

Comment:
I don't really like this bool pxd. Can you make it an enum instead? That makes it easier to understand the call site.

e.g. 

getPythonTypeName(myType, PYTHON_PXD)